Languages/en.ini

```diff
@@ -829,7 +829,7 @@ AllowFailingOutOfSet=If 'Yes' players can potentially fail out of a set early.
 NumberOfContinuesAllowed=Allow players to chain multiple games together by entering extra credits. Set this to 0 if you want to disable it.\n\nThis feature only applies to Pay Mode, because credits only exist in Pay Mode.
 MusicWheelSpeed=Change the speed at which the MusicWheel will scroll through content.\n\nA faster MusicWheel is especially helpful when the MenuTimer is enabled.
 
-AllowScreenSelectProfile=Set this to 'No' if you want to skip the Select Profile screen.\n\nThe Select Profile screen can be helpful if you have multiple local profiles.
+AllowScreenSelectProfile=Set this to 'No' if you want to skip the Select Profile screen and disable profile selection in the in-game Options menu.\n\nThe Select Profile screen can be helpful if you have multiple local profiles.

```

Comment:
I'm not sure if "in-game Options menu" is the canonical way to refer to it

Reply:
I'm not sure if I'd call it a Sort menu as it does all sorts of things and the menu header is "Options" - in fact, I think it does not non-sort stuff than sort stuff haha

how about "and disable profile switching via the Options Menu in the Select Music screen"

<img width="435" alt="Screenshot 2024-08-20 at 9 14 43 PM" src="https://github.com/user-attachments/assets/ccfa60f0-da0d-4b8b-a195-bb4a73112b64">

Reply:
Woops didn't respond but yeah I think that is fine. And good call about the Options header. I have to rewire my brain to not call it sort menu since that's actually what it's been called for a long time and I'm sure it's referenced as such throughout the code base.

Reply:
Sweet! I'll update this and get the translation in soon.